troposphere/codeartifact.py

```diff
@@ -24,6 +24,7 @@ class Repository(AWSObject):
 
     props = {
         'Description': (basestring, False),
+        'DomainName': (basestring, True),
```

Comment:
[`DomainOwner`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-codeartifact-repository.html#cfn-codeartifact-repository-domainowner) looks missing as well:
https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-codeartifact/pull/38#discussion_r518897362
```suggestion
        'DomainName': (basestring, True),
        'DomainOwner': (basestring, False),
```

Reply:
Interesting. I added the DomainName to a template that was rendered by Troposphere but did not provide DomainOwner and it worked in CloudFormation fine. You are right though, it should be added.